package.json

```diff
@@ -58,7 +58,10 @@
     "core-js": "^2.4.1",
     "eonasdan-bootstrap-datetimepicker": "^4.17.47",
     "fullcalendar": "^3.4.0",
+    "ical.js": "^1.2.2",
     "moment": "^2.18.1",
+    "request": "^2.81.0",
```

Comment:
Ah actually it appears that it is a [required peer dependency](https://github.com/request/request-promise#migration-from-v2-to-v3)